src/linear_algebra/matrix/hermitian.lean

```diff
@@ -9,6 +9,8 @@ import analysis.inner_product_space.pi_L2
 
 This file defines hermitian matrices and some basic results about them.
 
+See also `is_self_adjoint`, which generalizes this definition to other star rings.
```

Comment:
```suggestion
See also `is_self_adjoint`, which generalizes this definition to other star rings
in the case when the matrices are square.
```

Reply:
I don't quite follow; `is_hermitian` only exists in the case where the matrix is square anyway.

Reply:
Oh sorry, in my head I read `conj_transpose`.